modules/ROOT/pages/endpoint-sftp-receive-target.adoc

```diff
@@ -1,91 +1,72 @@
 = SFTP Receive Endpoint Settings
 
-Use these settings to set up SFTP endpoints in one of the following contexts:
+You can use these settings to set up SFTP endpoints in one of the following contexts:
```

Comment:
Do we need "in one of the following contexts"?
To me, it confused what the list I was going to see next actually listed: "these" settings or "the following" contexts.

Reply:
Should we spell out STFP in the intro text, or is your audience completely familiar with it?

Reply:
Done!

The audience would know what STFP is - it is a general file transfer process.